schemas/workertype-response.yml

```diff
@@ -0,0 +1,55 @@
+$schema:  http://json-schema.org/draft-04/schema#
+title:          "Worker-type Response"
+description: |
+  Response to a worker-type request from a provisioner.
+type:           object
+properties:
+  provisionerId:
+    title:        "Provisioner ID"
+    type:         string
+    minLength:    {$const: identifier-min-length}
+    maxLength:    {$const: identifier-max-length}
+    pattern:      {$const: identifier-pattern}
+  workerType:
+    type:         string
+    title:        "WorkerType name"
+    description: |
+      WorkerType name.
+    minLength:    {$const: identifier-min-length}
+    maxLength:    {$const: identifier-max-length}
+    pattern:      {$const: identifier-pattern}
+  stability:
+    title:        "Stability"
+    description: |
+      This is the stability of the worker-type. Accepted values:
+        * `experimental`
+        * `stable`
+        * `deprecated`
+    type:         string
+    enum:         ["experimental", "stable", "deprecated"]
+  description:
+    title:        "Description"
+    description: |
+      Description of the worker-type.
+    type:         string
+  expires:
+    title:        "Worker-type Expiration"
+    description: |
+      Date and time after which the worker-type will be automatically
+      deleted by the queue.
+    type:         string
+    format:       date-time
+  lastDateActive:
+    title:        "Worker-type Last Date Active"
+    description: |
+      Date and time where the worker-type was last seen active
```

Comment:
Let's add a note here similar to that for the lastDateUsed for clientIds in the auth API.  That says "Will only be updated every 6 hours or so this may be off by up-to 6 hours. But it still gives a solid hint as to whether or not this client is in use."  The grammar could be improved :)